credits.txt

```diff
@@ -61,3 +84,187 @@ Beta Testers
   Sean Fahey
   Mauricio Gomes
   Iaz Poolar
+
+Special thanks to MCOfficer for
+creating, managing, and hosting
+the nightly builds that make
+so much of this work possible.
+
+Proof Readers & Editors
+  Arrow2thekn33
+  Bladewood
+  Fzzr
+  jafdy
+  Janaszar
+  LocalGod79
+  MCOfficer
+  Moonbeam
+  RecursiveParadox
+  Tadrix
+  temtemy
+  Zearth Goh aka TheMarksman
+
+Story Creators
```

Comment:
Storyline & Faction Authors

Reply:
That's definitely a bit more clear. Changed!